src/Dropdown/Dropdown.js

```diff
@@ -82,7 +86,7 @@ export default class Dropdown extends React.Component {
           placement,
           modifiers: {
             offset: {
-              offset: '0, 10',
```

Comment:
Does this need to be `offset: this.props.offset`?

Reply:
Yes! That was the intention, thanks for catching the oversight, fixed.

Reply:
Was this tested locally? Browser would've thrown

Reply:
I hadn't tested it locally, is something like yalc the best tool for that? Or by running the docs command?

Reply:
Thanks for the reminder Kyle, screenshot added to description above. (I had been testing the changes locally by editing the molekule.es.js file in  cms/node_modules, and had forgotten to copy over the assignment.)

Reply:
Looks like you found `yarn docs` :)